exe/resqued

```diff
@@ -8,7 +8,7 @@ end
 
 require 'optparse'
 
-options = {}
+options = {:exec_on_hup => true}
```

Comment:
Should this use 1.9 hash style syntax?

Reply:
There's a bunch of 1.8 hash style syntax in resqued already, so I was trying to match what was already there. I'll change the new and updated code to use 1.9 syntax. If I don't forget to, I'll make a followup PR to change the rest of the hash syntax.